src/hotspot/share/cds/filemap.cpp

```diff
@@ -1589,13 +1591,16 @@ size_t FileMapInfo::remove_bitmap_leading_zeros(CHeapBitMap* map) {
     size_t new_zeros = map->find_first_set_bit(0);
     assert(new_zeros == 0, "Should have removed leading zeros");
   )
-
-  assert(map->size_in_bytes() < old_size, "Map size should have decreased");
+  assert(map->size() <= old_size, "Map size should have decreased");
```

Comment:
The comment is not correct, as the old size could be the same as the new size. Maybe just "sanity"?